pysgrid/tests/test_sgrid.py

```diff
@@ -398,22 +393,14 @@ def test_node_dimesnions(sgrid_wrf_sgrid_2d):
     assert node_dims == expected
 
 
-# TestSGridSaveNodeCoordinates
-"""
-Test that SGrid.save_as_netcdf is saving
-content correctly.
+# Test that SGrid.save_as_netcdf is saving
+# content correctly.
 
-There maybe a better way to do this using
-mocks, but this will do for now.
-
-"""
-
-
-def round_trip_deltares(deltares_sgrid):
+def test_round_trip_deltares(deltares_sgrid):
```

Comment:
I was wondering why they weren't called "test_*" -- and there was at least one typo in there, but they seemed to pass for me. We can remove them again if they don't seem trustworthy -- I didn't look closely to see if they did anything useful.